typescript/utils/class-color-generator.ts

```diff
@@ -0,0 +1,110 @@
+export const hexColorSequence = [
+  "#F87171",
+  "#FACC15",
+  "#34D399",
+  "#38BDF8",
+  "#A78BFA",
+  "#DB2777",
+  "#FB923C",
+  "#65A30D",
+  "#0D9488",
+  "#2563EB",
+  "#9333EA",
+  "#FB7185",
+  "#FBBF24",
+  "#4ADE80",
+  "#22D3EE",
+  "#818CF8",
+  "#E879F9",
+  "#DC2626",
+  "#CA8A04",
+  "#059669",
+  "#0284C7",
+  "#7C3AED",
+  "#F472B6",
+  "#EA580C",
+  "#A3E635",
+  "#2DD4BF",
+  "#60A5FA",
+  "#C084FC",
+  "#E11D48",
+  "#D97706",
+  "#16A34A",
+  "#0891B2",
+  "#4F46E5",
+  "#C026D3",
+];
+
+// It's charkra's gray.200
+export const noneClassColor = "#E2E8F0";
```

Comment:
Do we need to define a color for `none` at such a low-level?

Reply:
Not sure why, I kept the same definition

Reply:
I agree that the none color does not really belong here : this file is for generating colors for existing classes that will be stored, but the none color is not for a class, it's more in the UI design category, part of the site's style.

Reply:
I moved it